packages/ra-core/src/i18n/translate.js

```diff
@@ -33,9 +34,10 @@ const translate = BaseComponent => {
         translate: PropTypes.func.isRequired,
         locale: PropTypes.string.isRequired,
     };
-    TranslatedComponent.displayName = `TranslatedComponent(${
-        BaseComponent.displayName
-    })`;
+    TranslatedComponent.displayName = wrapDisplayName(
```

Comment:
I know this commit has nothing to see with the PR, but while testing the CloneButton with enzyme, I was really bothered by the `WithStyles(TranslatedComponent(undefined))` that I saw everywhere. Now they are `WithStyles(translate(Button))`, much better.

Reply:
It was on my to-do list too :+1: